doc/source/getting_started/faqs.rst

```diff
@@ -0,0 +1,235 @@
+.. _faqs:
+
+Frequently asked questions
+==========================
+
+What is PyAnsys?
+----------------
+PyAnsys is a set of open source technologies that allow you to interface Pythonically
+with Ansys Fluent, Mechanical APDL, AEDT, and other Ansys products. You can use PyAnsys
+libraries within a Python environment of your choice in conjunction with external Python
+libraries.
+
+.. image:: ../_static/PyAnsys_overview.png
+  :width: 800
+  :alt: PyAnsys overview
+
+What is PyFluent?
+-----------------
+PyFluent provides Pythonic access to Ansys Fluent. Its features enable the seamless use of
+Fluent within the Python ecosystem and broad access to native Fluent features for performing
+actions such as these:
+
+- Launch Fluent using a local Ansys installation.
```

Comment:
@PipKat @dnwillia-work I realise that this is also the current wording. The fact that one can also connect to a remote instance seems worth a mention here.

Reply:
Maybe propose a bullet point wording.

Reply:
I added another bullet about connecting to a Fluent instance running on a remote machine.